templates/telemetry/crash/crash.4.schema.json

```diff
@@ -70,6 +70,10 @@
             }
           }
         },
+        "processType": {
+          "type": "string",
+          "enum": [ "main", "content", "gpu" ]
```

Comment:
Do we want the ping to fail validation if we introduce a new `processType`?  If there's no strong reason to make this an `enum`, I would somewhat prefer to leave this as a free-form string value.

Reply:
Good point, I'll leave it as free-form string.

Reply:
+1, a free-form string value would be more flexible in the future.
Is there any potential down-side to allowing free-form values, e.g. would it affect analysis in some way if unexpected values show up?

Reply:
I checked some of the existing data and it is well-behaved (one of these three values or null), so I'm more worried about intentionally-introduced new values (maybe `dynamic`?) than a bunch of unexpected values.

If we start to observe unexpected values here it seems like we should address it at that time.

Reply:
But how would you know you're getting unexpected values if it's not validated?
On the other hand I totally agree with the flexibility point.

Reply:
You'd start to see them appear in datasets / analyses downstream.

Reply:
I'm kind of the opinion that we should have just included all current known process types [0]. But as per usual I'm late to the party :)

[0] https://searchfox.org/mozilla-central/source/toolkit/components/crashes/nsICrashService.idl#24

Reply:
I'm not confident enough in our alerting on validation failures yet, so I'm inclined to be permissive / future-proof where possible.

Reply:
@mreid-moz that is concerning, can you make a bug to track that?

Reply:
I think we should prioritize that bug highly so we don't have to be concerned with missing data, and ending up with too lenient schemas.